etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/CreateRawBaseTables.java

```diff
@@ -278,16 +273,20 @@ public void executeStageLoad(String[] params) {
     }
 
 
+    private String getName(String name){
+        return "SHOW TABLES LIKE '" + name + "'";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SHOW TABLES LIKE '" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)